test/compilable/json.d

```diff
@@ -2,10 +2,6 @@
 // REQUIRED_ARGS: -d -preview=dip1000 -o- -X -Xf${RESULTS_DIR}/compilable/json.out
 // POST_SCRIPT: compilable/extra-files/json-postscript.sh
 // EXTRA_FILES: imports/jsonimport1.d imports/jsonimport2.d imports/jsonimport3.d imports/jsonimport4.d
-/* TEST_OUTPUT:
```

Comment:
This test seems to be failing for some reason.

Reply:
@ibuclaw The issue is that removing those comments is going to affect the output of the json in terms of line numbers and therefore the expected output of this test also needs to be modifed.

Reply:
It's weird that circleCI passes though.